vllm/engine/arg_utils.py

```diff
@@ -820,18 +826,31 @@ def create_engine_config(self, ) -> EngineConfig:
             disable_logprobs=self.disable_logprobs_during_spec_decoding,
         )
 
+        if self.num_scheduler_steps > 1:
```

Comment:
can you also raise it when chunked prefill is enabled?

Reply:
a36ce22